sirepo/raydata/analysis_driver/chx.py

```diff
@@ -46,9 +42,19 @@ def get_output_dir(self):
             self.uid,
         )
 
+    def _get_papermill_args(self, *args, **kwargs):
+        return [
+            ["run_two_time", True],
+            ["run_dose", False],
+            ["username", self._scan_metadata.get_start_field("user")],
+        ]
+
 
 _cfg = pkconfig.init(
     base_dir=pkconfig.Required(
         pkio.py_path, "base directory for notebooks and outputs"
     ),
+    conda_env=pkconfig.RequiredUnlessDev(
```

Comment:
No need for RequiredUnlessDev. This is the name of the conda env at BNL.

Reply:
a4175ab0127a5b2207443ee18cab9d384c401eb9